web/containers/admin/GenericApplicantContainer/index.js

```diff
@@ -146,7 +148,17 @@ class GenericApplicantContainer extends React.Component {
       return undefined;
     }
     // this.setState({ applicantIndex: firestore.data[shortInfoCollectionName] });
-    const applicants = Object.values(firestore.data[shortInfoCollectionName]);
+    let applicants = Object.values(firestore.data[shortInfoCollectionName]);
+
+    if (searchText !== '') {
```

Comment:
The search is working great, awesome work!

Reply:
Thanks Cham!!!